app/src/main/java/org/oppia/android/app/topic/TopicFragmentPresenter.kt

```diff
@@ -105,10 +106,12 @@ class TopicFragmentPresenter @Inject constructor(
         }
       }
     }.attach()
-    if (topicId.isNotEmpty() && storyId.isNotEmpty())
-      setCurrentTab(TopicTab.LESSONS)
-    else if (topicId.isNotEmpty() && storyId.isEmpty())
-      setCurrentTab(TopicTab.INFO)
+    if (!isConfigChanged) {
+      if (topicId.isNotEmpty() && storyId.isNotEmpty())
+        setCurrentTab(TopicTab.LESSONS)
+      else if (topicId.isNotEmpty() && storyId.isEmpty())
+        setCurrentTab(TopicTab.INFO)
```

Comment:
Suggest using a ``when`` instead. Also, be aware that per https://developer.android.com/kotlin/style-guide#formatting we should always add braces if any statements span multiple lines.

Reply:
Should i change these lines to 
```
    when(isConfigChanged) {
      false -> {
        if(topicId.isNotEmpty() && storyId.isNotEmpty()) {
          setCurrentTab(TopicTab.LESSONS)
        }
      }
    }
```
because  by default the viewpager opens topic info fragment in the viewpager.

Reply:
I was actually thinking this:

```kotlin
    if (!isConfigChanged) {
      when {
        topicId.isNotEmpty() && storyId.isNotEmpty() -> setCurrentTab(TopicTab.LESSONS)
        topicId.isNotEmpty() && storyId.isEmpty() -> setCurrentTab(TopicTab.INFO)
      }
    }
```

However, that made me then realize this whole thing can be simplified a bit more:

```kotlin
    if (!isConfigChanged && topicId.isNotEmpty()) {
      setCurrentTab(if (storyId.isNotEmpty()) TopicTab.LESSONS else TopicTab.INFO)
    }
```

I'm fine with any of these options (the two above or the current implementation).

Reply:
The above approach makes the code look much cleaner. changed the code with as the above approach